app/helpers/displayHelpers.php

```diff
@@ -2690,16 +2690,16 @@ function caCreateLinksFromText($pa_text, $ps_table_name, $pa_row_ids, $ps_class=
 		if (!$g_request) { return $pa_text; }
 		
 		foreach($pa_text as $vn_i => $vs_text) {
-			$vs_text = preg_replace("!([A-Za-z0-9]+)='([^']*)'!", "$1=\"$2\"", $vs_text);	
+			$vs_text = preg_replace("!([A-Za-z0-9]+)='([^']*)'!", "$1=\"$2\"", $vs_text);
 			$va_l_tags = array();
 			$o_links = $o_doc('l');
-			
+
 			foreach($o_links as $o_link) {
 				if (!$o_link) { continue; }
 				$vs_html = $o_link->html();
 				
-				$vs_content = preg_replace("!^<[^\>]+>!", "", $vs_html);
```

Comment:
For the record, the sequence `\>` is an invalid escape sequence in a PHP string (double-quoted).  It's looking for something like `\n` but `\>` isn't valid.  I think what was intended was `\\>` which translates to `\>` in the regular expression, but the `>` character doesn't need to be escaped in this context.  That's why I removed the `\`.

However I've backed this out because it's not really related to the fix, it was just me stopping the IDE from complaining.